packages/emails/src/components/Separator.tsx

```diff
@@ -0,0 +1,3 @@
+export const Separator = () => (
+  <p style={{ width: "16px", height: "16px", display: "inline-block" }}>&nbsp;</p>
+);
```

Comment:
New separator component for email in this case to separate two buttons from each other